spec/services/file_manager_spec.rb

```diff
@@ -7,27 +7,15 @@
   let(:service_token) { SecureRandom.hex }
   let(:service_slug) { 'service-slug' }
   let(:encrypted_user_id_and_token) { SecureRandom.hex(16) }
-
-  subject do
-    described_class.new(encoded_file: encoded_file,
-                        user_id: user_id,
-                        service_slug: service_slug,
-                        encrypted_user_id_and_token: encrypted_user_id_and_token)
-  end
-
-  describe '#key' do
```

Comment:
Removed this as its a private method. The object it calls is also tested separately.